src/antsibull/__init__.py

```diff
@@ -1,2 +1,4 @@
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
Why is this wrapped? Is that some linting-related thing?

Reply:
Yes, there's a maximum line length of 100 configured everywhere.